### PR TITLE
Add missing French translation of tutoriel_pipeline.md

### PR DESCRIPTION
**Pull request by jadechoghari**
# What does this PR do?

This pull request adds a French translation of the `pipeline_tutorial` from English to French.

## Changes made
- Translated `pipeline_tutorial.md` from English to French.
- Updated the `_toctree.yml` to include the French version of the pipeline tutorial.

## Why is this change important?
Providing documentation in multiple languages helps make the information accessible to a broader audience and supports users who are more comfortable reading in French. 🇫🇷
More docs to come. 🤗

## Review
Like last time I will tag:
@amyeroberts (for the merge ✅)
@younesbelkada (for the review 🇫🇷)

### CODE REVIEW

File: docs/source/fr/tutoriel_pipeline.md [hunk 1]
```diff
@@ -0,0 +1,314 @@
+<!--⚠️ Note that this file is in Markdown but contain specific syntax for our doc-builder (similar to MDX) that may not be
+rendered properly in your Markdown viewer.
+-->
+
+# Pipelines pour l'inférence
+
+Le [`pipeline`] rend simple l'utilisation de n'importe quel modèle du [Hub](https://huggingface.co/models) pour l'inférence sur n'importe quelle langue, tâches de vision par ordinateur, de parole et multimodales. Même si vous n'avez pas d'expérience avec une modalité spécifique ou si vous n'êtes pas familier avec le code ci-dessous des modèles, vous pouvez toujours les utiliser pour l'inférence avec le [`pipeline`] ! Ce tutoriel vous apprendra à :
```
Comment (younesbelkada) [inline docs/source/fr/tutoriel_pipeline.md:7]:
```suggestion
L'objet [`pipeline`] rend simple l'utilisation de n'importe quel modèle du [Hub](https://huggingface.co/models) pour l'inférence sur n'importe quelle langue, tâches de vision par ordinateur, d'audio et multimodales. Même si vous n'avez pas d'expérience avec une modalité spécifique ou si vous n'êtes pas familier avec le code ci-dessous des modèles, vous pouvez toujours les utiliser pour l'inférence avec la [`pipeline`] ! Ce tutoriel vous apprendra à :
```
I would say here `paroles` -> `audio` and would use `une` pipeline

File: docs/source/fr/tutoriel_pipeline.md [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+
+* Utiliser un [`pipeline`] pour l'inférence.
```
Comment (younesbelkada) [inline docs/source/fr/tutoriel_pipeline.md:9]:
```suggestion
* Utiliser une [`pipeline`] pour l'inférence.
```

File: docs/source/fr/tutoriel_pipeline.md [hunk 3]
```diff
@@ -0,0 +1,314 @@
...
+* Utiliser un tokenizer ou modèle spécifique.
+* Utiliser un [`pipeline`] pour des tâches audio, de vision et multimodales.
```
Comment (younesbelkada) [inline docs/source/fr/tutoriel_pipeline.md:11]:
```suggestion
* Utiliser une [`pipeline`] pour des tâches audio, de vision et multimodales.
```

File: docs/source/fr/tutoriel_pipeline.md [hunk 4]
```diff
@@ -0,0 +1,314 @@
...
+
+<Tip>
+
+Consultez la documentation du [`pipeline`] pour une liste complète des tâches prises en charge et des paramètres disponibles.
+
+</Tip>
+
+## Utilisation du pipeline
+
+Bien que chaque tâche ait son propre [`pipeline`], il est plus simple d'utiliser le [`pipeline`] générale qui inclut tous les pipelines spécifiques aux différentes tâches. Cette approche charge automatiquement un modèle par défaut et une classe de prétraitement adaptée à votre tâche, simplifiant ainsi votre utilisation. Prenons l'exemple de l'utilisation du [`pipeline`] pour la reconnaissance automatique de la parole (ASR) ou de la transcription de la parole en texte.
```
Comment (younesbelkada) [inline docs/source/fr/tutoriel_pipeline.md:21]:
```suggestion
Bien que chaque tâche ait son propre [`pipeline`], il est plus simple d'utiliser la [`pipeline`] générale qui inclut tous les pipelines spécifiques aux différentes tâches. Cette approche charge automatiquement un modèle par défaut et une classe de prétraitement adaptée à votre tâche, simplifiant ainsi votre utilisation. Prenons l'exemple de l'utilisation du [`pipeline`] pour la reconnaissance vocale (ASR) ou de la transcription de la parole en texte.
```
`reconnaissance automatique de la parole` -> `reconnaissance vocale`

File: docs/source/fr/tutoriel_pipeline.md [hunk 5]
```diff
@@ -0,0 +1,314 @@
...
+
+1. Commencez par créer un [`pipeline`] et spécifiez la tâche d'inférence :
```
Comment (younesbelkada) [inline docs/source/fr/tutoriel_pipeline.md:23]:
```suggestion
1. Commencez par créer une [`pipeline`] et spécifiez la tâche d'inférence :
```

File: docs/source/fr/tutoriel_pipeline.md [hunk 6]
```diff
@@ -0,0 +1,314 @@
...
+
+```py
+>>> from transformers import pipeline
+
+>>> transcriber = pipeline(task="automatic-speech-recognition")
+```
+
+2. Passez votre entrée au [`pipeline`]. Dans le cas de la reconnaissance vocale, il s'agit d'un fichier audio :
```
Comment (younesbelkada) [inline docs/source/fr/tutoriel_pipeline.md:31]:
```suggestion
2. Passez votre entrée à la [`pipeline`]. Dans le cas de la reconnaissance vocale, il s'agit d'un fichier audio :
```

File: docs/source/fr/tutoriel_pipeline.md [hunk 7]
```diff
@@ -0,0 +1,314 @@
...
+```py
+>>> transcriber("https://huggingface.co/datasets/Narsil/asr_dummy/resolve/main/mlk.flac")
+{'text': 'I HAVE A DREAM BUT ONE DAY THIS NATION WILL RISE UP LIVE UP THE TRUE MEANING OF ITS TREES'}
+```
+
+Pas le résultat que vous aviez en tête ? Consultez certains des [modèles de reconnaissance vocale automatique les plus téléchargés](https://huggingface.co/models?pipeline_tag=automatic-speech-recognition&sort=trending) 
+sur le Hub pour voir si vous pouvez obtenir une meilleure transcription.
+
+Essayons le modèle [Whisper large-v2](https://huggingface.co/openai/whisper-large) de OpenAI. Whisper a été publié 
+2 ans après Wav2Vec2 et a été entraîné sur près de 10 fois plus de données. En tant que tel, il surpasse Wav2Vec2 sur la plupart des benchmarks en aval. Il a également l'avantage supplémentaire de prédire la ponctuation et la casse, ce qui n'est pas possible avec Wav2Vec2.
```
Comment (younesbelkada) [inline docs/source/fr/tutoriel_pipeline.md:42]:
```suggestion
Essayons le modèle [Whisper large-v2](https://huggingface.co/openai/whisper-large) de OpenAI. Whisper a été publié 2 ans après Wav2Vec2 et a été entraîné sur près de 10 fois plus de données. En tant que tel, il surpasse Wav2Vec2 sur la plupart des benchmarks en aval. Il a également l'avantage supplémentaire de prédire la ponctuation et la casse, ce qui n'est pas possible avec Wav2Vec2.
```

File: docs/source/fr/tutoriel_pipeline.md [hunk 8]
```diff
@@ -0,0 +1,314 @@
...
+```py
+transcriber(
+    [
+        "https://huggingface.co/datasets/Narsil/asr_dummy/resolve/main/mlk.flac",
+        "https://huggingface.co/datasets/Narsil/asr_dummy/resolve/main/1.flac",
+    ]
+)
+```
+
+Les pipelines sont excellents pour l'expérimentation car passer d'un modèle à un autre est trivial ; cependant, il existe des moyens de les optimiser pour des charges de travail plus importantes que l'expérimentation. Consultez les guides suivants qui expliquent comment itérer sur des ensembles de données complets ou utiliser des pipelines dans un serveur web :
```
Comment (younesbelkada) [inline docs/source/fr/tutoriel_pipeline.md:68]:
```suggestion
Les pipelines sont excellentes pour l'expérimentation car passer d'un modèle à un autre est trivial ; cependant, il existe des moyens de les optimiser pour des charges de travail plus importantes que l'expérimentation. Consultez les guides suivants qui expliquent comment itérer sur des ensembles de données complets ou utiliser des pipelines dans un serveur web :
```

File: docs/source/fr/tutoriel_pipeline.md [hunk 9]
```diff
@@ -0,0 +1,314 @@
...
+de la documentation :
+* [Utilisation des pipelines sur un ensemble de données](#using-pipelines-on-a-dataset)
+* [Utilisation des pipelines pour un serveur web](./pipeline_webserver)
+
+## Paramètres
+
+[`pipeline`] prend en charge de nombreux paramètres ; certains sont spécifiques à la tâche et d'autres sont généraux pour tous les pipelines.
```
Comment (younesbelkada) [inline docs/source/fr/tutoriel_pipeline.md:75]:
```suggestion
[`pipeline`] prend en charge de nombreux paramètres ; certains sont spécifiques à la tâche et d'autres sont généraux pour toutes les pipelines.
```

File: docs/source/fr/tutoriel_pipeline.md [hunk 10]
```diff
@@ -0,0 +1,314 @@
...
+transcriber = pipeline(model="openai/whisper-large-v2", my_parameter=1)
+
+out = transcriber(...)  # This will use `my_parameter=1`.
+out = transcriber(..., my_parameter=2)  # This will override and use `my_parameter=2`.
+out = transcriber(...)  # This will go back to using `my_parameter=1`.
+```
+
+Voyons 3 paramètres importants :
+
+### Appareil
```
Comment (younesbelkada) [inline docs/source/fr/tutoriel_pipeline.md:88]:
```suggestion
### Device
```
I would keep `device` here

File: docs/source/fr/tutoriel_pipeline.md [hunk 11]
```diff
@@ -0,0 +1,314 @@
...
+
+Si vous utilisez `device=n`, le pipeline met automatiquement le modèle sur l'appareil spécifié.
```
Comment (younesbelkada) [inline docs/source/fr/tutoriel_pipeline.md:90]:
```suggestion
Si vous utilisez `device=n`, le pipeline met automatiquement le modèle sur le device spécifié.
```

File: docs/source/fr/tutoriel_pipeline.md [hunk 12]
```diff
@@ -0,0 +1,314 @@
...
+
+Le code suivant charge et stocke automatiquement les poids du modèle sur plusieurs appareils :
+
+```py
+transcriber = pipeline(model="openai/whisper-large-v2", device_map="auto")
+```
+
+Notez que si `device_map="auto"` est passé, il n'est pas nécessaire d'ajouter l'argument `device=device` lors de l'instanciation de votre `pipeline` car vous pourriez rencontrer des comportements inattendus !
+
+### Taille de lot
```
Comment (younesbelkada) [inline docs/source/fr/tutoriel_pipeline.md:111]:
```suggestion
### Batch size
```
I also wouldn't translate batc size here and keep it as is

File: docs/source/fr/tutoriel_pipeline.md [hunk 13]
```diff
@@ -0,0 +1,314 @@
...
+
+Par défaut, les pipelines ne feront pas de lot d'inférence pour des raisons expliquées en détail [ici](https://huggingface.co/docs/transformers/main_classes/pipelines#pipeline-batching). La raison est que le lotissement n'est pas nécessairement plus rapide, et peut en fait être beaucoup plus lent dans certains cas.
```
Comment (younesbelkada) [inline docs/source/fr/tutoriel_pipeline.md:113]:
```suggestion
Par défaut, les pipelines ne feront pas d'inférence en batch pour des raisons expliquées en détail [ici](https://huggingface.co/docs/transformers/main_classes/pipelines#pipeline-batching). La raison est que le batching n'est pas nécessairement plus rapide, et peut en fait être beaucoup plus lent dans certains cas.
```

File: docs/source/fr/tutoriel_pipeline.md [hunk 14]
```diff
@@ -0,0 +1,314 @@
...
+
+Mais si cela fonctionne dans votre cas d'utilisation, vous pouvez utiliser :
+
+```py
+transcriber = pipeline(model="openai/whisper-large-v2", device=0, batch_size=2)
+audio_filenames = [f"https://huggingface.co/datasets/Narsil/asr_dummy/resolve/main/{i}.flac" for i in range(1, 5)]
+texts = transcriber(audio_filenames)
+```
+
+Cela exécute le pipeline sur les 4 fichiers audio fournis, mais les passera par lots de 2 au modèle (qui est sur un GPU, où le lotissement est plus susceptible d'aider) sans nécessiter de code supplémentaire de votre part.
```
Comment (younesbelkada) [inline docs/source/fr/tutoriel_pipeline.md:123]:
```suggestion
Cela exécute le pipeline sur les 4 fichiers audio fournis, mais les passera par batch de 2 au modèle (qui est sur un GPU, où le batching est plus susceptible d'aider) sans nécessiter de code supplémentaire de votre part. 
```

File: docs/source/fr/tutoriel_pipeline.md [hunk 15]
```diff
@@ -0,0 +1,314 @@
...
+Mais si cela fonctionne dans votre cas d'utilisation, vous pouvez utiliser :
+
+```py
+transcriber = pipeline(model="openai/whisper-large-v2", device=0, batch_size=2)
+audio_filenames = [f"https://huggingface.co/datasets/Narsil/asr_dummy/resolve/main/{i}.flac" for i in range(1, 5)]
+texts = transcriber(audio_filenames)
+```
+
+Cela exécute le pipeline sur les 4 fichiers audio fournis, mais les passera par lots de 2 au modèle (qui est sur un GPU, où le lotissement est plus susceptible d'aider) sans nécessiter de code supplémentaire de votre part. 
+La sortie doit toujours correspondre à ce que vous auriez reçu sans lotissement. Il s'agit uniquement d'un moyen de vous aider à obtenir plus de vitesse avec un pipeline.
```
Comment (younesbelkada) [inline docs/source/fr/tutoriel_pipeline.md:124]:
```suggestion
La sortie doit toujours correspondre à ce que vous auriez reçu sans batching. Il s'agit uniquement d'un moyen de vous aider à obtenir plus de vitesse avec un pipeline.
```

File: docs/source/fr/tutoriel_pipeline.md [hunk 16]
```diff
@@ -0,0 +1,314 @@
...
+
+Les pipelines peuvent également atténuer certaines des complexités du lotissement car, pour certains pipelines, un seul élément (comme un long fichier audio) doit être divisé en plusieurs parties pour être traité par un modèle. Le pipeline effectue ce [*lotissement par morceaux*](./main_classes/pipelines#pipeline-chunk-batching) pour vous.
```
Comment (younesbelkada) [inline docs/source/fr/tutoriel_pipeline.md:126]:
```suggestion
Les pipelines peuvent également atténuer certaines des complexités du batching car, pour certains pipelines, un seul élément (comme un long fichier audio) doit être divisé en plusieurs parties pour être traité par un modèle. Le pipeline effectue ce [*batching par morceaux*](./main_classes/pipelines#pipeline-chunk-batching) pour vous.
```

File: docs/source/fr/tutoriel_pipeline.md [hunk 17]
```diff
@@ -0,0 +1,314 @@
...
+...     "I have a problem with my iphone that needs to be resolved asap!!",
+...     candidate_labels=["urgent", "not urgent", "phone", "tablet", "computer"],
+... )
+{'sequence': 'I have a problem with my iphone that needs to be resolved asap!!', 'labels': ['urgent', 'phone', 'computer', 'not urgent', 'tablet'], 'scores': [0.504, 0.479, 0.013, 0.003, 0.002]}
+```
+
+
+## Pipeline multimodal
+
+Le [`pipeline`] prend en charge plus d'une modalité. Par exemple, une tâche de réponse à des questions visuelles (VQA) combine texte et image. N'hésitez pas à utiliser n'importe quel lien d'image que vous aimez et une question que vous souhaitez poser à propos de l'image. L'image peut être une URL ou un chemin local vers l'image.
```
Comment (younesbelkada) [inline docs/source/fr/tutoriel_pipeline.md:237]:
```suggestion
La [`pipeline`] peut prend en charge plus d'une modalité. Par exemple, une tâche de réponse à des questions visuelles (VQA) combine texte et image. N'hésitez pas à utiliser n'importe quel lien d'image que vous aimez et une question que vous souhaitez poser à propos de l'image. L'image peut être une URL ou un chemin local vers l'image.
```

File: docs/source/fr/tutoriel_pipeline.md [hunk 18]
```diff
@@ -0,0 +1,314 @@
...
+## Création de démonstrations web à partir de pipelines avec `gradio`
+
+Hugging Face prenant en charge le chargement de grands modèles, comme BLOOM.
+Les pipelines sont automatiquement pris en charge dans [Gradio](https://github.com/gradio-app/gradio/), une bibliothèque qui facilite la création d'applications d'apprentissage automatique belles et conviviales sur le web. Tout d'abord, assurez-vous que Gradio est installé :
+
+```
+pip install gradio
+```
+
+Ensuite, vous pouvez créer une démonstration web autour d'un pipeline de classification d'images (ou tout autre pipeline) en une seule ligne de code en appelant la fonction [`Interface.from_pipeline`](https://www.gradio.app/docs/interface#interface-from-pipeline) de Gradio pour lancer le pipeline. Cela crée une interface intuitive de glisser-déposer dans votre navigateur :
```
Comment (younesbelkada) [inline docs/source/fr/tutoriel_pipeline.md:300]:
```suggestion
Ensuite, vous pouvez créer une démonstration web autour d'une pipeline de classification d'images (ou tout autre pipeline) en une seule ligne de code en appelant la fonction [`Interface.from_pipeline`](https://www.gradio.app/docs/interface#interface-from-pipeline) de Gradio pour lancer le pipeline. Cela crée une interface intuitive de "glisser-déposer" (*Drag and Drop* en anglais) dans votre navigateur :
```

**Review by younesbelkada — COMMENTED** [review]
Thanks a lot for this contribution ! I left some suggestions - what do you think?

File: docs/source/fr/tutoriel_pipeline.md (hunk 2 above)
Comment (jadechoghari) [reply docs/source/fr/tutoriel_pipeline.md:9]: Apparemment:
Définition de pipeline ​​​ | ​​​ nom masculin
https://dictionnaire.lerobert.com/definition/pipeline#:~:text=D%C3%A9finition%20de%20pipeline%20%E2%80%8B%E2%80%8B,%E2%9E%99%20gazoduc%2C%20ol%C3%A9oduc.

**Comment by jadechoghari** [comment]
Great :) !, will just leave the pipeline in masculine form.
Should I resolve?
Thanks !

File: docs/source/fr/tutoriel_pipeline.md (hunk 11 above)
Comment (jadechoghari) [reply docs/source/fr/tutoriel_pipeline.md:89]: Device -> appareil (correct? 🇫🇷)
https://www.linguee.com/english-french/translation/device.html

File: docs/source/fr/tutoriel_pipeline.md (hunk 17 above)
Comment (jadechoghari) [reply docs/source/fr/tutoriel_pipeline.md:236]: yes - adding peut is good

**Comment by younesbelkada** [comment]
Sounds great to leave that, thanks for explaining !

**Review by younesbelkada — APPROVED** [review]
Thanks a lot ! 🚀

**Review by amyeroberts — APPROVED** [review]
Thanks!

**Pull request merged**